packages/amplify-category-api/src/provider-utils/awscloudformation/cdk-stack-builder/types.ts

```diff
@@ -34,7 +34,7 @@ type AmplifyCDKL1 = {
   addCfnCondition(props: cdk.CfnConditionProps, logicalId: string): void;
   addCfnMapping(props: cdk.CfnMappingProps, logicalId: string): void;
   addCfnOutput(props: cdk.CfnOutputProps, logicalId: string): void;
-  addCfnParameter(props: cdk.CfnParameterProps, logicalId: string): void;
+  addCfnParameter(props: cdk.CfnParameterProps, logicalId: string, value?: any): void;
```

Comment:
I guess we can discuss on this to return the cdk obj also with these functions.
```
  addCfnResource(props: cdk.CfnResourceProps, logicalId: string): CfnResource;
```

Reply:
I'm open to expanding the API, but I think that's better for another PR. This change was just fixing a type that I forgot to update in a previous PR.